prov/rxd/src/rxd_cq.c

```diff
@@ -985,6 +989,8 @@ static void rxd_handle_ack(struct rxd_ep *ep, struct rxd_pkt_entry *ack_entry)
 	fi_addr_t peer = ack->base_hdr.peer;
 	struct rxd_base_hdr *hdr;
 
+	ep->peers[peer].tx_window = ack->ext_hdr.rx_id;
```

Comment:
It's common to make windows relative to a specific sequence number (e.g. seq_no=1000, window=16, window is for packets 1001-1016, or specify the window as being in same range as the sequence number (e.g. seq_no = 1000, window = 1016.  Absolute values cause issues when handling duplicate or out of order messages.

I don't know if the code has an issue, but the above assignment is suspect.